$NetBSD: patch-aw,v 1.4 2014/08/25 08:23:22 wiz Exp $

Use CPPFLAGS from environment.
Already included in current Advanced release.

--- lib/Makefile.orig	Fri Sep 18 19:04:37 2009
+++ lib/Makefile	Fri Aug  3 13:52:47 2012
@@ -81,7 +81,7 @@
 $(LIBOBJECTS): %.o: %.c importinc
 # Note that the user may have configured -I options into CFLAGS.
 	$(CC) -c $(INCLUDES) -DNDEBUG $(CFLAGS) $(CFLAGS_SHLIB) \
-	  $(CFLAGS_PERSONAL) $(CADD) -o $@ $<
+	  $(CFLAGS_PERSONAL) $(CPPFLAGS) $(CADD) -o $@ $<
 
 MAJ = $(NETPBM_MAJOR_RELEASE)
 MIN = $(NETPBM_MINOR_RELEASE)
@@ -205,11 +205,11 @@
 .PHONY: install.lib
 ifeq ($(NETPBMLIBTYPE),unixshared)
 # install a Unix-style shared library
-install.lib: $(PKGDIR)/lib $(PKGDIR)/link
-	cd $(PKGDIR)/lib ; rm -f libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ).*
+install.lib: $(PKGDIR)/lib$(LIBARCHSUFFIX) $(PKGDIR)/link
+	cd $(PKGDIR)/lib$(LIBARCHSUFFIX) ; rm -f libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ).*
 	$(INSTALL) -c -m $(INSTALL_PERM_LIBD) \
-	  libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ).$(MIN)  $(PKGDIR)/lib/
-	cd $(PKGDIR)/lib/ ; \
+	  libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ).$(MIN)  $(PKGDIR)/lib$(LIBARCHSUFFIX)
+	cd $(PKGDIR)/lib$(LIBARCHSUFFIX) ; \
           rm -f libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ); \
           $(SYMLINK) libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ).$(MIN) $(SONAME)
 endif
@@ -257,7 +257,7 @@
 # install the link-time (.so) links to the runtime libraries
 	cd $(PKGDIR)/link ; \
           rm -f libnetpbm.$(NETPBMLIBSUFFIX); \
-          $(SYMLINK) ../lib/libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ) \
+          $(SYMLINK) libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ) \
             libnetpbm.$(NETPBMLIBSUFFIX)
 endif
 ifeq ($(NETPBMLIBTYPE),dll)
